src/styles/blocks/product.scss

```diff
@@ -0,0 +1,120 @@
+.product-card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  width: $product-card-width;
+
+  border: 1px solid $product-border-color;
+  border-radius: $product-border-radius;
+
+  &__image {
+    box-sizing: border-box;
+    background-image: url('../images/imac.jpeg');
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__content {
+    margin: 40px 16px 16px;
+
+    font-family: $product-font-family, sans-serif;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $product-main-color;
+    margin: 0;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: product-secondary-color;
```

Comment:
There seems to be a typo here: `product-secondary-color` should be `$product-secondary-color`. Ensure all SCSS variables are prefixed with a `$` to avoid compilation errors.